data-model/src/main/java/io/micronaut/data/model/query/builder/AbstractSqlLikeQueryBuilder.java

```diff
@@ -698,9 +734,10 @@ protected void buildSelect(AnnotationMetadata annotationMetadata, QueryState que
             selectAllColumns(annotationMetadata, queryState, queryString);
         } else {
             int projectionCount = projectionList.size();
-            for (Iterator i = projectionList.iterator(); i.hasNext(); ) {
+            for (Iterator<QueryModel.Projection> i = projectionList.iterator(); i.hasNext(); ) {
                 boolean appendComma = true;
-                QueryModel.Projection projection = (QueryModel.Projection) i.next();
+                boolean booleanRemoveComma = false;
```

Comment:
Probably named by mistake, var should be renamed from `booleanRemoveComma` to `removeComma' I guess.